test/docfx.Test/lib/JsonSchemaTest.cs

```diff
@@ -435,5 +471,32 @@ public void TestJsonSchemaPostValidation(string schema, string[] jsons, int erro
             var errors = jsonSchemaValidator.PostValidate();
             Assert.Equal(errorCount, errors.Count);
         }
+
+        [Theory]
+        [InlineData(
+            "{'properties':{ 'key':{'properties': {'key1': {'dateFormat': 'M/d/yyyy'}}}}}",
+            "{'key': {'key1': 'Dec 5 2018'}}",
+            "{'message_severity':'warning','code':'date-format-invalid','message':'Invalid date format for 'key.key1': 'Dec 5 2018'.','file':'file','line':1,'end_line':1,'column':29,'end_column':29}")]
+
+        public void TestJsonSchemaValidation2(string schema, string json, string expectedErrors)
```

Comment:
What is the different between this method and `TestJsonSchemaValidation`?

Reply:
Oh, this is for my test case and I'll delete it.